mir_robots/mir_bringup/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ find_package(catkin REQUIRED)
 
 catkin_package(
   CATKIN_DEPENDS
-    hokuyo_node
+#    hokuyo_node
```

Comment:
`hokuyo_node` has been replaced by `urg_node`

Reply:
Changed to urg_node.